src/features/deepLinking/__tests__/deepLinking.test.ts

```diff
@@ -1,5 +1,5 @@
+import { BrowserWindowMock } from '../../../../__mocks__/browser-window'
```

Comment:
You don't need to import this any more even, jest will automock it for you. Just import it directly from electron. The import will be automatically modified to BrowserWindow from `__mocks__` directory

Reply:
Updated